src/lib.rs

```diff
@@ -15,6 +13,7 @@
 
 use getopts;
 
+#[cfg(feature = "unstable")]
 extern crate test;
```

Comment:
Will this cause breakage when the crate is built as part of the sysroot because `test` isn't available?

Reply:
i'm not sure, ideally the parts necessary for that will be moved to libcore `mem::` or `hint::black_box` modules, and then we;d avoid that